tests/php/test-amp-helper-functions.php

```diff
@@ -1254,6 +1254,161 @@ static function ( $script ) {
 		$this->assertStringContains( '<script type=\'text/javascript\' src=\'https://cdn.ampproject.org/v0/amp-foo-0.1.js\' async custom-element="amp-foo"></script>', $output ); // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
 	}
 
+	/** @covers ::amp_register_default_scripts() */
+	public function test_amp_register_default_scripts() {
```

Comment:
Note that all of the versions being checked in this test are the same as in `develop`, with the exception of the three _new_ components added:

* `amp-cache-url`
* `amp-iframely`
* `amp-stream-gallery`

So there will be no unexpected component version changes if this is part of a 2.1.3 patch release.